src/main/resources/db/migration/V6_Add_frozen_user.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "user" ADD COLUMN "frozen" boolean NOT NULL DEFAULT FALSE;
```

Comment:
This file needs 2 underscores between `V6` and `Add`:

`V6_Add_frozen_user.sql -> V6__Add_frozen_user.sql`